omni-relayer/src/utils/near.rs

```diff
@@ -103,66 +103,33 @@ pub async fn handle_streamer_message(
                 utils::redis::add_event(
                     redis_connection,
                     utils::redis::NEAR_SIGN_TRANSFER_EVENTS,
-                    message_payload.nonce.0.to_string(),
+                    message_payload.destination_nonce.to_string(),
                     log,
                 )
                 .await;
             }
             Nep141LockerEvent::FinTransferEvent {
-                ref nonce,
                 ref transfer_message,
             } => {
-                if nonce.is_some() {
-                    utils::redis::add_event(
-                        redis_connection,
-                        utils::redis::NEAR_INIT_TRANSFER_QUEUE,
-                        transfer_message.origin_nonce.0.to_string(),
-                        crate::workers::near::InitTransferWithTimestamp {
-                            event: log,
-                            creation_timestamp: chrono::Utc::now().timestamp(),
-                            last_update_timestamp: None,
-                        },
-                    )
-                    .await;
-                } else {
-                    utils::redis::add_event(
-                        redis_connection,
-                        utils::redis::NEAR_SIGN_CLAIM_NATIVE_FEE_QUEUE,
-                        transfer_message.origin_nonce.0.to_string(),
-                        log,
-                    )
-                    .await;
-                }
+                utils::redis::add_event(
+                    redis_connection,
+                    utils::redis::NEAR_SIGN_CLAIM_NATIVE_FEE_QUEUE,
```

Comment:
We shouldn't have `NEAR_SIGN_CLAIM_NATIVE_FEE_QUEUE` because we don't have sign_claim_native_fee function

Reply:
Well, I completely removed `sign_claim_native_fee` logic in https://github.com/Near-One/omni-bridge/pull/146/commits/8126804275e90842252e8e047083e91fcfbaa10d. But now I'm not sure what should I do on "FinTransferEvent" on near side
https://github.com/Near-One/omni-bridge/blob/76e65901ad53c77cf6eaafc97b573a60a7f0f42e/omni-relayer/src/utils/near.rs#L111-L113